lib/ruby_indexer/lib/ruby_indexer/declaration_listener.rb

```diff
@@ -805,6 +808,45 @@ def handle_module_function(node)
       end
     end
 
+    sig { params(node: Prism::CallNode).void }
+    def handle_private_class_method(node)
+      node.arguments&.arguments&.each do |argument|
+        string_or_symbol_nodes = case argument
+        when Prism::StringNode, Prism::SymbolNode
+          [argument]
+        when Prism::ArrayNode
+          argument.elements
+        else
+          []
+        end
+
+        unless string_or_symbol_nodes.empty?
+          # pop the visibility off since there isn't a method definition following `private_class_method`
+          @visibility_stack.pop
```

Comment:
I needed to pop this off the stack otherwise subsequent class method definitions would also get the private visibility.